### ISSUE TITLE
Does not work (Possibly Intellij Idea related)

### ISSUE THREAD

**Issue by AkashaP** [TravaOpenJDK/trava-jdk-11-dcevm#26]
I'm getting errors after upgrading to dcevm 11.0.8+1, can't build my project, it was working before i upgraded versions

Not sure if related but i also updated Kotlin version to 1.4.0-rc-release-IJ2020.1-2

It's says loading hotswap agent {1.4.1} and then crashes with
Stacktrace
```
Starting HotswapAgent '/home/akasha/.jdks/dcevm-11.0.8+1/lib/hotswap/hotswap-agent.jar'
HOTSWAP AGENT: 02:19:49.544 INFO (org.hotswap.agent.HotswapAgent) - Loading Hotswap agent {1.4.1} - unlimited runtime class redefinition.
Exception in thread "main" java.lang.reflect.InvocationTargetException
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:513)
        at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallPremain(InstrumentationImpl.java:525)
Caused by: java.lang.NullPointerException
        at org.hotswap.agent.config.PluginManager.init(PluginManager.java:138)
        at org.hotswap.agent.HotswapAgent.premain(HotswapAgent.java:69)
        ... 6 more
*** java.lang.instrument ASSERTION FAILED ***: "result" with message agent load/premain call failed at ./src/java.instrument/share/native/libinstrument/JPLISAgent.c line: 422
FATAL ERROR in native method: processing of -javaagent failed, processJavaStart failed
Aborted (core dumped)
```

strange... getting the same error when downgrading back to 11.0.7+4 now.

The full running command (linux, using Intellij Idea) is
```
/home/__user__/.jdks/dcevm-11.0.8+1/bin/java -Xmx700m -Djdt.compiler.useSingleThread=true -Dexternal.project.config=/home/__user__/.cache/JetBrains/IdeaIC2020.1/external_build_system/velandia.74c99177 -Dcompile.parallel=false -Drebuild.on.dependency.change=true -Djava.net.preferIPv4Stack=true -Dio.netty.initialSeedUniquifier=-1847999843681461108 -Dfile.encoding=UTF-8 -Duser.language=en -Duser.country=AU -Didea.config.path=/home/__user__/.config/JetBrains/IdeaIC2020.1 -Didea.plugins.path=/home/__user__/.local/share/JetBrains/IdeaIC2020.1 -Djps.log.dir=/home/__user__/.cache/JetBrains/IdeaIC2020.1/log/build-log -Djps.fallback.jdk.home=/home/__user__/Applications/idea-IC-192.6603.28/jbr -Djps.fallback.jdk.version=11.0.7 -Dio.netty.noUnsafe=true -Djava.io.tmpdir=/home/__user__/.cache/JetBrains/IdeaIC2020.1/compile-server/velandia_4b8b998d/_temp_ -Djps.backward.ref.index.builder=true -Dkotlin.incremental.compilation=true -Dkotlin.incremental.compilation.js=true -Dkotlin.daemon.enabled -Dkotlin.daemon.client.alive.path=\"/tmp/kotlin-idea-13035598240147056381-is-running\" -classpath /home/__user__/Applications/idea-IC-192.6603.28/plugins/java/lib/jps-launcher.jar org.jetbrains.jps.cmdline.Launcher /home/__user__/Applications/idea-IC-192.6603.28/lib/oro-2.0.8.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/httpclient-4.5.12.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/commons-lang3-3.9.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/guava-28.2-jre.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/idea_rt.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/qdox-2.0-M10.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/commons-logging-1.2.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-resolver-spi-1.3.3.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/platform-api.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/netty-buffer-4.1.47.Final.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/lz4-java-1.7.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-builder-support-3.6.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/netty-resolver-4.1.47.Final.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/netty-codec-4.1.47.Final.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-resolver-provider-3.6.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-repository-metadata-3.6.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/log4j.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/forms-1.1-preview.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/asm-all-7.0.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/slf4j-api-1.7.25.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/jdom.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/java/lib/maven-resolver-transport-http-1.3.3.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/nanoxml-2.2.3.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/java/lib/jps-builders.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-artifact-3.6.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/jna-platform.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/netty-common-4.1.47.Final.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/protobuf-java-3.5.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/java/lib/javac2.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/netty-transport-4.1.47.Final.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/java/lib/maven-resolver-connector-basic-1.3.3.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/annotations.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/httpcore-4.4.13.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-resolver-api-1.3.3.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/java/lib/jps-builders-6.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-model-3.6.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/util.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/gson-2.8.6.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-model-builder-3.6.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/jps-model.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/plexus-component-annotations-1.7.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/java/lib/aether-dependency-resolver.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/java/lib/maven-resolver-transport-file-1.3.3.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/plexus-interpolation-1.25.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/jna.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/resources_en.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-resolver-impl-1.3.3.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/trove4j.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/maven-resolver-util-1.3.3.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/plexus-utils-3.2.0.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/gradle/lib/gradle-api-6.1.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/gradle/lib/gradle-api-impldep-6.1.1.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/ant/lib/ant.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/groovy-all-2.4.17.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/gson-2.8.6.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/slf4j-api-1.7.25.jar:/home/__user__/Applications/idea-IC-192.6603.28/lib/slf4j-log4j12-1.7.25.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/IntelliLang/lib/intellilang-jps-plugin.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/eclipse/lib/eclipse-jps-plugin.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/eclipse/lib/common-eclipse-util.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/Groovy/lib/groovy-jps-plugin.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/Groovy/lib/groovy-rt-constants.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/maven/lib/maven-jps-plugin.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/gradle-java/lib/gradle-jps-plugin.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/devkit/lib/devkit-jps-plugin.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/javaFX/lib/javaFX-jps-plugin.jar:/home/__user__/Applications/idea-IC-192.6603.28/plugins/javaFX/lib/common-javaFX-plugin.jar:/home/__user__/.local/share/JetBrains/IdeaIC2020.1/Kotlin/lib/jps/kotlin-jps-plugin.jar:/home/__user__/.local/share/JetBrains/IdeaIC2020.1/Kotlin/lib/kotlin-stdlib.jar:/home/__user__/.local/share/JetBrains/IdeaIC2020.1/Kotlin/lib/kotlin-reflect.jar:/home/__user__/.local/share/JetBrains/IdeaIC2020.1/Kotlin/lib/kotlin-plugin.jar org.jetbrains.jps.cmdline.BuildMain 127.0.0.1 38259 6e8d3685-c0b9-4332-8b7f-27bca932edc4 /home/__user__/.cache/JetBrains/IdeaIC2020.1/compile-server

```

**Comment by AkashaP** [comment]
Downgrading Intellij Idea to 2020.1.3 seems to have fixed the problem. Perhaps the fix for 2020.1.4 over at [https://youtrack.jetbrains.com/issue/IDEA-240241](url), which seems to be related to JRebel Hotswapagent, broke this dvecm Hotswapagent??

**Comment by skybber** [comment]
Could you please check it with https://github.com/TravaOpenJDK/trava-jdk-11-dcevm/releases/tag/dcevm-11.0.9%2B2 ?
